src/test/jtx/AMM.h

```diff
@@ -106,18 +106,6 @@ struct VoteArg
     std::optional<ter> err = std::nullopt;
 };
 
-struct BidArg
```

Comment:
I'd like to keep this. *Arg and functions that use it is a preferred way moving forward for AMM unit-tests.